src/content/guides/prepare-aws-account-for-tenant-clusters/index.md

```diff
@@ -80,16 +80,16 @@ These are the limit increases to be requested, grouped by limit type:
 When requesting a service limit increase, you will be asked for a description of your use case. You can use this text for the purpose:
 
 > We intend to run multiple Kubernetes clusters in this account, potentially used
-by various globally distributed teams. We will be creating and deleting new
+by various roles within globally distributed teams. We will be creating and deleting new
```

Comment:
```suggestion
by various globally distributed teams. We will be creating and deleting new
```

this is not about roles, this is about teams using clusters

Reply:
I added roles here since the context is IAM. So it's not just the teams themselves, it's even different people on those teams.
If that doesn't make sense, I will adopt your suggestion.

Reply:
It's not about IAM as our customers do not use IAM to authenticate against Kubernetes, IAM is used in a limited manner and only by components within the infrastructure. The sentence here is to explain the need for many separate clusters in an account (VPC limit increases for example)